sdk/storage/azure-storage-blob-nio/README.md

```diff
@@ -0,0 +1,215 @@
+# Azure Storage Blob client library for Java
```

Comment:
Azure Storage Blob NIO client library

Reply:
Everything in the README I'm leaving to work on later. But I'll add a note at the top that clarifies that